querydsl-codegen/src/main/java/com/querydsl/codegen/UncapitalizedCaseTransformer.java

```diff
@@ -0,0 +1,19 @@
+package com.querydsl.codegen;
+
+import com.mysema.codegen.StringUtils;
+
+/**
+ * Uncapitalizes the first letter of a name.
+ *
+ */
+public class UncapitalizedCaseTransformer implements CaseTransformer {
+
+    /**
+     * @see com.querydsl.codegen.CaseTransformer#transform(java.lang.String)
+     */
+    @Override
+    public String transform(String name) {
```

Comment:
Yes, that looks good.
What about the instatiation of the variableNameFunction? Would you inject the Function in the constructor?